README.md

```diff
@@ -20,17 +20,26 @@ See [Releasing a new version of the Sling starter](https://cwiki.apache.org/conf
         directory. It is advisable to use a work directory outside of the
         project directory.
 
-1) Build the Sling Starter using 
+1) Build the Sling Starter using `mvn clean install` in the current directory.
+2) Copy the launch to project directory `cp target/dependency/org.apache.sling.feature.launcher.jar $path-to-project`
```

Comment:
Any particular reason to add this step? I think it's simpler to not copy it, but use it directly in its configured location.